index.js

```diff
@@ -36,6 +36,19 @@ function stack(results) {
 
 		i++;
 
+		// Don't print the full stack but the only useful line showing
+		// the actual test file stack
+		var reg = /\((?:[\\\/](?:(?!node_modules[\\\/]ava[\\\/])[^:\\\/])+)+:\d+:\d+\)/;
+		function beautifulStack(stack) {
+			var found = false;
+			return stack.split('\n').filter(function (line) {
+				var relevant = reg.test(line);
+				found = found || relevant;
+				return !found || relevant;
+			}).join('\n');
+		}
+		result.error.stack = beautifulStack(result.error.stack);
```

Comment:
Something makes me feel iffy about re-assigning it back to the error itself. Thoughts @sindresorhus?